.github/workflows/release-integration.yml

```diff
@@ -58,34 +58,36 @@ jobs:
       DOCKER_PLATFORMS: "linux/amd64,linux/arm64,linux/arm" # Must be consistent with the matrix from the job above.
     steps:
       - uses: actions/checkout@v2
+      - uses: docker/setup-qemu-action@v1
+      - uses: docker/setup-buildx-action@v1
+      - uses: actions/download-artifact@v2
+      - uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.FSI_DOCKERHUB_USERNAME }}
+          password: ${{ secrets.FSI_DOCKERHUB_TOKEN }}
 
       - name: Generate Docker image version from git tag
         run: |
           echo "${{ github.event.release.tag_name }}" | grep -E '^v[0-9.]*[0-9]$'
           DOCKER_IMAGE_TAG=$(echo "${{ github.event.release.tag_name }}" | sed 's/^v//')
           echo "DOCKER_IMAGE_TAG=$DOCKER_IMAGE_TAG" >> $GITHUB_ENV
-      - if: ${{ github.event.release.prerelease }}
+
+      - name: Change Docker image version if this run is a pre-release
+        if: ${{ github.event.release.prerelease }}
         run: |
           echo "DOCKER_IMAGE_TAG=${DOCKER_IMAGE_TAG}-pre" >> $GITHUB_ENV
-      - uses: docker/setup-qemu-action@v1
-      - uses: docker/setup-buildx-action@v1
-
-      - name: Download all artifacts from build job
-        uses: actions/download-artifact@v2
-
-      - uses: docker/login-action@v1
-        with:
-          username: ${{ secrets.FSI_DOCKERHUB_USERNAME }}
-          password: ${{ secrets.FSI_DOCKERHUB_TOKEN }}
 
       - name: Build and push Docker image
+        if: ${{ github.event.release.prerelease }}
         run: |
           docker buildx build --push --platform=$DOCKER_PLATFORMS \
             -t $DOCKER_IMAGE_NAME:$DOCKER_IMAGE_TAG \
             .
-      - name: Push :latest image
```

Comment:
in the K8s pipeline we have before pushing:
```
      - name: Run Trivy vulnerability scanner
        uses: aquasecurity/trivy-action@0.2.2
        with:
          image-ref: '${{ env.DOCKER_IMAGE_NAME }}:ci-scan'
          format: 'table'
          exit-code: '0'
          ignore-unfixed: true
          severity: 'CRITICAL,HIGH'
```

Should we add it here as well?

Reply:
one year ago I [changed](https://github.com/newrelic/nri-kubernetes/commit/f8ad2e4d766a9ca86d92d226fcd3849c02d0222f) to '0' and we forgot that like that since then

Reply:
I think it would be best if the security job happened on a different workflow file, with different rules. I'd leave it out of the scope for this one.

Reply:
I would add it for the same reason @kang-makes mentioned, but I am also fine with leaving it like this

Reply:
This is starting to take too much time. I am going to leave it for the day we will meet to plan the next quarter.